src/app/submission/objects/submission-objects.effects.ts

```diff
@@ -201,14 +201,13 @@ export class SubmissionObjectEffects {
       return this.operationsService.jsonPatchByResourceType(
         this.submissionService.getSubmissionObjectLinkName(),
         action.payload.submissionId,
-        'sections').pipe(
+        'sections',
+        true).pipe(
```

Comment:
I'd avoid to send a patch request with an empty body when there is nothing to save. I'd rather do a simple get request of the submission object. 
To determinate if there are changes to save you can use the [hasUnsavedModification method](https://github.com/DSpace/dspace-angular/blob/main/src/app/submission/submission.service.ts#L442)